### PR TITLE
Adds phobia quirk.

### PR DESCRIPTION
**Pull request by super12pl**
## About The Pull Request
Adds negative phobia quirk which at cost of -2 points gives the holder random permanent phobia.

## Why It's Good For The Game
Phobias are fun, let's have more of them

## Changelog
:cl:
add: Phobia quirk
/:cl:

### CODE REVIEW

**Comment by SergeiKoralev** [comment]
If there is a way to list the specific phobia in the medical records that would be based as hell.

File: code/datums/traits/negative.dm [hunk 1]
```diff
@@ -646,3 +646,16 @@
...
+
+/datum/quirk/phobia
+	name = "Phobia"
+	desc = "Because of a traumatic event in your past you have developed a strong phobia."
+	value = -2
+	gain_text = "<span class='danger'>You start feeling an irrational fear of something.</span>"
+	lose_text = "<span class='notice'>You are no longer irrationally afraid.</span>"
+	medical_record_text = "Patient suffers from a deeply-rooted phobia."
+
+/datum/quirk/paraplegic/add()
```
Comment (St0rmC4st3r) [inline code/datums/traits/negative.dm]: Why are you so cruel towards paraplegic people? What did they do to you?

Comment (St0rmC4st3r) [inline code/datums/traits/negative.dm]: Remover also needed.

**Review by St0rmC4st3r — CHANGES_REQUESTED** [review]

File: code/datums/traits/negative.dm (hunk 1 above)
Comment (super12pl) [reply code/datums/traits/negative.dm]: Added

**Review by Archanial — APPROVED** [review]

**Comment by qwertyquerty** [comment]
Is this really worth 2 points?

**Comment by St0rmC4st3r** [comment]
> Is this really worth 2 points?

Considering how crippling some phobias can be (security, doctors), yes

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**